src/Core/Domain/Discount/Command/UpdateDiscountConditionsCommand.php

```diff
@@ -95,4 +98,50 @@ public function setProductConditions(array $productConditions): self
 
         return $this;
     }
+
+    public function setConditionApplicationType(string $conditionApplicationType): self
+    {
+        $this->conditionApplicationType = $conditionApplicationType;
+
+        return $this;
+    }
+
+    public function setConditionType(string $conditionType): self
+    {
+        $this->conditionType = $conditionType;
+
+        return $this;
+    }
+
+    public function getConditionApplicationType(): ?string
+    {
+        return $this->conditionApplicationType;
+    }
+
+    public function getConditionType(): ?string
+    {
+        return $this->conditionType;
+    }
```

Comment:
I don't think we need all this, fromwhat I remember this is not persisted anywhere This is an information that is mostly related to the way the form is built And it can be deduced based on the associated product rules dynamically, so it could be computed in the `FormDataProvider` just so we know which component needs to be shown by default